internal/cmd/organization/command_describe.go

```diff
@@ -0,0 +1,44 @@
+package organization
+
+import (
+	"github.com/spf13/cobra"
+
+	pcmd "github.com/confluentinc/cli/internal/pkg/cmd"
+	"github.com/confluentinc/cli/internal/pkg/errors"
+	"github.com/confluentinc/cli/internal/pkg/output"
+	"github.com/confluentinc/cli/internal/pkg/resource"
+)
+
+type out struct {
+	IsCurrent bool   `human:"Current" serialized:"is_current"`
+	Id        string `human:"ID" serialized:"id"`
+	Name      string `human:"Name" serialized:"name"`
+}
+
+func (c *command) newDescribeCommand() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "describe",
+		Short: "Describe the current Confluent Cloud organization.",
+		Args:  cobra.NoArgs,
+		RunE:  c.describe,
+	}
+
+	pcmd.AddOutputFlag(cmd)
```

Comment:
`--context`?

Reply:
I'm on the fence. For describe it wouldn't matter too much, but I think it might be bad for update because the org itself is not part of the context, so a user could end up updating the name of the wrong org.

Reply:
> a user could end up updating the name of the wrong org

Organizations have unique resource IDs, right? I don't think that should be a problem. The `--context` flag would allow you to describe or update an organization without needing to log in to that org again, seems useful to me!

Reply:
They have unique resource id's, but the resource id is not part of the context name. I just did a test using a pair of stag accounts; the first is in 2 orgs. I logged into the second and then used the first user's context for this flag and the org that got updated depended on whichever one the first user last logged in to.

Because it's not in the name, you can't choose and you have to call describe with that context or look in the config json file to see which org will be updated.

Reply:
> the org that got updated depended on whichever one the first user last logged in to

Ah, I see. Sounds like we don't want the `--context` flag for any of the `confluent organization` commands, then.

Reply:
Yeah it's unfortunate. Is adding the org to the context something we could consider doing, maybe for v4? That would resolve this.

Reply:
Hmm, the organization should be able to be inferred from the context, right?

Reply:
Yeah it's stored in the context.

Reply:
Yeah, can you make a ticket for that? It looks like right now if a user is in more than one organization, their old context will get overridden by a new one when the log in to a different organization... not good!

Reply:
Will do, and that sounds like the behavior I was seeing when I was testing this.